src/main/java/gift/service/WishService.java

```diff
@@ -0,0 +1,50 @@
+package gift.service;
+
+import gift.dto.ProductResponseDto;
+import gift.dto.WishResponseDto;
+import gift.repository.WishRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Transactional
+@Service
+public class WishService {
+
+    public final WishRepository wishRepository;
+
+    public WishService(WishRepository wishRepository) {
+        this.wishRepository = wishRepository;
+    }
+
+    public void saveWish(long memberId, long productId, int count) {
+        if (count == 0)
+            return;
+
+        wishRepository.saveWish(memberId, productId, count);
```

Comment:
```suggestion
        if (count == 0){
            return;
        }

        wishRepository.saveWish(memberId, productId, count);
```
if()문을 쓸때는 중괄호를 써주는게 가독성 측면에서 더 좋을 것 같아요!
save시도하기 전에 검증해서 미리 리턴해준게 성능적으로 좋네요👍

Reply:
넵 알겠습니다1